### PR TITLE
Added support for destroy_value [Issues#66]

### PR DESCRIPTION
**Pull request by mmroz**

### CODE REVIEW

**Comment by mmroz** [comment]
Issue: https://github.com/themaplelab/swift/issues/66

File: lib/WALASupport/SILWalaInstructionVisitor.cpp [hunk 1]
```diff
@@ -1073,6 +1073,23 @@ jobject SILWalaInstructionVisitor::visitCopyValueInst(CopyValueInst *CVI) {
...
+
+  ValueBase *key = static_cast<ValueBase *>(Value);
+  if (NodeMap.find(key) != NodeMap.end()) {
+    if (Print) {
+      llvm::outs() << "\t\t Value " << Value.getOpaqueValue() <<  " destroyed \n";
+    }
+    NodeMap.erase(key);
+  }
+
+  return Node;
```
Comment (karimhamdanali) [inline lib/WALASupport/SILWalaInstructionVisitor.cpp]: why does the method return Node instead of nullptr?

**Comment by mmroz** [comment]
Good catch!

**Comment by karimhamdanali** [comment]
all the credit goes to @taufique71  :)

**Review by karimhamdanali — APPROVED** [review]

**Pull request merged**